### PR TITLE
ci: automatically merge master to develop

### PR DESCRIPTION
**Pull request by barmac**
This merges master to develop automatically and notifies us via Slack if merge fails.

### CODE REVIEW

**Comment by barmac** [comment]
This is how it works in action: https://github.com/barmac/test/runs/2991786270?check_suite_focus=true

File: .github/workflows/MERGE_MASTER_TO_DEVELOP.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        git config user.name github-actions
+        git config user.email github-actions@github.com
+        git merge -m 'Merge master to develop' --no-edit origin/master
+        git push
+    - name: Notify on Slack if merge fails
+      uses: slackapi/slack-github-action@v1.15.0
+      if: failure()
+      with:
+        channel-id: ${{ secrets.SLACK_CHANNEL_ID }}
+        slack-message: "Could not merge <https://github.com/${{ github.repository }}/tree/${{ github.ref }}|${{ github.ref }}> to <https://github.com/${{ github.repository }}/tree/develop|${{ github.repository }}#develop>."
```
Comment (nikku) [inline .github/workflows/MERGE_MASTER_TO_DEVELOP.yml:30]: Proposal: Rename to something more actionable:

```
Automatic merge of <https://github.com/${{ github.repository }}/tree/${{ github.ref }}|${{ github.ref }}> to <https://github.com/${{ github.repository }}/tree/develop|${{ github.repository }}#develop failed.
```

**Review by nikku — COMMENTED** [review]
Looks great!

What is our story for re-using such shared behaviors?

I believe GitHub has a thing that we could use?

File: .github/workflows/MERGE_MASTER_TO_DEVELOP.yml (hunk 1 above)
Comment (barmac) [reply .github/workflows/MERGE_MASTER_TO_DEVELOP.yml:30]:
```suggestion
        slack-message: "Automatic merge of <https://github.com/${{ github.repository }}/tree/${{ github.ref }}|${{ github.ref }}> to <https://github.com/${{ github.repository }}/tree/develop|${{ github.repository }}#develop failed."
```

**Comment by barmac** [comment]
> What is our story for re-using such shared behaviors?

Copy and paste the workflow to a selected repository. Alternatively, the second step could be rewritten as an action which could be used elsewhere.

**Review by nikku — APPROVED** [review]
:clap:

**Comment by nikku** [comment]
Copy and paste, I :heart: it!

**Pull request merged**